### PR TITLE
SonarCloud code smell minors

### PR DESCRIPTION
**Pull request by cartoon-face**
This should clear up all the code smell minor warnings on SonarCloud

### CODE REVIEW

**Comment by krypt0x** [comment]
Nice one! Waiting for one more reviewer to approve. Thanks

**Review by krypt0x — APPROVED** [review]
Nice work! All seems fine. Thanks

**Review by AxVultis — APPROVED** [review]
All good. Thanks for your work!

**Pull request merged**
